api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -224,6 +224,9 @@ message HttpService {
 
   // Settings used for controlling authorization response metadata.
   AuthorizationResponse authorization_response = 8;
+
+  // Specifies the conditions under which request should be failed.
+  repeated string failed_on = 9;
```

Comment:
I think repeated enum is still better. But yeah we can wait for the API review later.